consul_kv_migrator.py

```diff
@@ -0,0 +1,97 @@
+from consul_kv import Connection
+import json
+import ast
+from prettytable import PrettyTable
+import getopt
+import sys
+
+argv = sys.argv[1:]
+
+def _emptyCheckerFunc(argv):
+  if not argv:
+    print("use -h tag to get more information about the utility")
+    exit()
+
+_emptyCheckerFunc(argv)
+
+try:
+  opts, args = getopt.getopt(argv, "s:d:o:h")
+
+except getopt.GetoptError:
+  print("USAGE:")
+  print("-h:  To get more information about the script options")
+  print("FORMAT: python script.py -h")
+  print("")
+  exit()
+
+
+for opt, arg in opts:
+
+  if opt in ['-s']:
+    _sourceConsulLink = arg
+  if opt in ['-d']:
+    _destinationConsulLink = arg
+  if opt in ['-o']:
+    _outputType = arg
+  elif opt in ['-h']:
+    print("Usage: python <NAME-OF-SCRIPT>.py ... [ -<OPTIONS> <VALUES>]")
+    print("")
+    print("Mendatory Option and arguments")
+    print("")
+    print("-s : Provide source consul URL")
+    print("     FORMAT: -s <VALUE>")
+    print("")
+    print("-d : Provide destination consul URL")
+    print("     FORMAT: -d <VALUE>")
+    print("")
+    print("-o : Provide database hostname")
+    print("")
+    print("     FORMAT: -o <VALUE>")
+    print("     SUPPORTED TAGS:")
+    print("         - json")
+    print("         - table")
+    print("")
+    print("")
+    print("AUTHOR: https://github.com/b44rawat")
+    print("")
+    sys.exit()
+
+try:
+    _sourceConsulObjectVar = Connection(endpoint=_sourceConsulLink)
+    _destinationConsulObjectVar = Connection(endpoint=_destinationConsulLink)
+except:
+    print('Check with -h tag to get more detail about utils options')
```

Comment:
I guess this statement in print is not valid for try block ..  You should fetch sepecif exception here ..maybe timeout or connection refused exception